vcf2hl7v2/test/vcf_example2.vcf

```diff
@@ -0,0 +1,423 @@
+##fileformat=VCFv4.1
+#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	NA12878
+10	96448001	.	T	<CGA_CNVWIN>	.	.	NS=1;CGA_WINEND=96450000	GT:CGA_GP:CGA_NP:CGA_CP:CGA_PS:CGA_CT:CGA_TS:CGA_CL:CGA_LS	.:0.87:0.92:2:53:=:53:0.999:707
+10	96448129	.	A	G	.	.	NS=1;AN=2;AC=1;CGA_XR=dbsnp.92|rs1926706;CGA_FI=1562|NM_000772.2|CYP2C18|INTRON|UNKNOWN-INC&1562|NM_001128925.1|CYP2C18|INTRON|UNKNOWN-INC;CGA_RPT=L1MA4A|L1|18.5	GT:PS:FT:GQ:HQ:EHQ:CGA_CEHQ:GL:CGA_CEGL:DP:AD:CGA_RDP	1/0:.:PASS:238:355,238:354,238:53,50:-355,0,-238:-53,0,-50:34:18,16:16
+10	96448380	.	C	G	.	.	NS=1;AN=2;AC=1;CGA_XR=dbsnp.120|rs11815200;CGA_FI=1562|NM_000772.2|CYP2C18|INTRON|UNKNOWN-INC&1562|NM_001128925.1|CYP2C18|INTRON|UNKNOWN-INC	GT:PS:FT:GQ:HQ:EHQ:CGA_CEHQ:GL:CGA_CEGL:DP:AD:CGA_RDP	1/0:.:PASS:247:247,247:241,241:47,50:-247,0,-247:-47,0,-50:37:13,24:24
+10	96448986	.	TTTAA	.	.	.	NS=1;AN=0	GT:PS	./.:.
+10	96448994	.	A	C	.	.	NS=1;AN=2;AC=1;CGA_FI=1562|NM_000772.2|CYP2C18|INTRON|UNKNOWN-INC&1562|NM_001128925.1|CYP2C18|INTRON|UNKNOWN-INC;CGA_RPT=AT_rich|Low_complexity|4.7	GT:PS:FT:GQ:HQ:EHQ:CGA_CEHQ:GL:CGA_CEGL:DP:AD:CGA_RDP	1|0:96448994:VQLOW:24:24,29:12,17:0,22:-24,0,-29:0,0,-22:8:1,7:7
+10	96449112	.	TTTCTATATTTTTATGAA	.	.	.	NS=1;AN=0	GT:PS	./.:.
+10	96449153	.	GTGGTCCTAGAAGGGACTGCTGC	.	.	.	NS=1;AN=0	GT:PS	./.:.
+10	96449429	.	A	.	.	.	NS=1;AN=0	GT:PS	./.:.
+10	96450001	.	T	<CGA_CNVWIN>	.	.	NS=1;CGA_WINEND=96452000	GT:CGA_GP:CGA_NP:CGA_CP:CGA_PS:CGA_CT:CGA_TS:CGA_CL:CGA_LS	.:0.82:1.1:.:0:.:0:0.999:707
+10	96450204	.	ATTTCTCGACA	.	.	.	NS=1;AN=0	GT:PS	./.:.
```

Comment:
Can we take a smaller vcf files for tests? Larger files increase the repository size unnecessarily.
Our test vcf file should have only one row for each format, having a similar format multiple records would not make difference in the test and increase the size of vcf file.

Reply:
Hi @srgothi92, Actually these vcf files are the exact same ones we use in `vcf2fhir` for the **integration tests**. Most of the files are already very small (less than 100 KB, some even less than 50 KB). For example, the file `vcf_example2.vcf` is 46 KB. Removing some of the records from this file will not make much of a difference. The largest file in `vcf2hl7v2` is `geneKB_Gene_table.csv` which is 2 MB alone but it cannot be reduced as it is required as it is for the program.